src/workerd/api/kv-instrumentation-test.js

```diff
@@ -8,6 +8,7 @@ import * as assert from 'node:assert';
 // each
 let invocationPromises = [];
 let spans = new Map();
+let topLevelSpanIds = new Map(); // Track top-level span IDs per invocation
```

Comment:
Non-blocking: Pass the top-level spanId directly to the tail worker, that way we don't need to create a map for them. (for reference, `src/workerd/api/tail-worker-test.js` already does this)